Add recipe for dkl

flexibeast opened the pull request:
### Brief summary of what the package does

Provide an ASCII-art representation of a keyboard layout, within an Emacs buffer. 

### Direct link to the package repository

https://github.com/flexibeast/dkl

### Your association with the package

Maintainer.

### Relevant communications with the upstream package maintainer

None.

### Checklist

- [x] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [x] I've built and installed the package using the instructions in the [README](https://github.com/melpa/melpa/blob/master/README.md)

syohex commented:
I have sent PR(https://github.com/flexibeast/dkl/pull/1)

purcell commented:
Thanks - looks interesting! Quick feedback:
- Your `dkl.el` still says `display-keyboard-layout.el` in it: this _must_ match the package name.
- You need to declare a package dependency on `(cl-lib "0.5")`
- Please use `:defaults` in the `:files` list instead of naming the `dkl.el` file individually

flexibeast commented:
@purcell, @syohex:

Sorry for the inconvenience! i've now:

* changed the reference to `display-keyboard-layout.el` to `dkl.el`;
* changed the recipe to use `:defaults`;
* removed the dependency on `cl-lib`, by using `(car (cddr ...` in place of `cl-caddr`.

purcell commented:
Great, thanks.
